update/ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -159,6 +159,13 @@
           tags:
             - skip_ansible_lint # '503 Tasks that run when changed should likely be handlers'.
       when: is_database_sqlite.stat.exists
+
+    - name: Change default admin id
```

Comment:
IMO, this is unnecessary during the initialization phase, PMM is going to run the update part down below on every restart or upgrade.

Reply:
I answered in the other PR that this will run only on upgrade, because of condition below for the whole block

Reply:
https://github.com/percona/pmm/pull/2531#discussion_r1361121886

Reply:
I disagree, but commit :)